docs/pages/get-started/installation.md

```diff
@@ -2,52 +2,51 @@
 title: Installation
 ---
 
-There are two tools that you need to develop apps with Expo: a local development tool and a mobile client to open your app.
+import TerminalBlock from '~/components/plugins/TerminalBlock';
+
+Developing Expo apps locally requires a CLI, and a client app or web browser to run the project. You can also get started in the browser with [Snack](https://snack.expo.io/).
 
 ## 1. Local development tool: Expo CLI
 
-Expo CLI is a tool for developing apps with Expo. In addition to the command-line interface (CLI) it also has a web-based graphical user interface (GUI) that pops up in your web browser when you start your project &mdash; you can use this if you're not yet comfortable with the using terminal or just prefer GUIs, both have similar capabilities.
+Expo CLI is a tool for developing apps and websites with React. Expo CLI also has a web-based GUI that pops up in your web browser when you start your project &mdash; this can be used in favor of the CLI if you're not yet comfortable using a terminal or just prefer GUIs, both have similar capabilities.
 
-### Pre-requisities
+### Requirements
 
-- **Node.js**: In order to install Expo CLI you will need to have Node.js (**we recommend the latest stable version**- but the maintenance and active LTS releases will also work) installed on your computer. [Download the recommended version of Node.js](https://nodejs.org/en/).
-- **Git**: Additionally, you'll need Git to create new projects. [You can download Git from here](https://git-scm.com).
+- [Node.js 12](https://nodejs.org/en/) or greater
+- [Git](https://git-scm.com) for version control
+- [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall) for MacOS users
 
 ### Installing Expo CLI
 
-We recommend installing Expo CLI globally, you can do this by running the following command:
-
-```
-npm install -g expo-cli
-```
-
-Verify that the installation was successful by running `expo whoami`. You're not logged in yet, so you will see "Not logged in". You can create an account by running `expo register` if you like, or if you have one already run `expo login`, but you also don't need an account to get started.
+The best way to get started with universal React is by using `expo-cli`, alternatively you can use [`create-react-native-app`](https://github.com/expo/create-react-native-app) for more templates.
 
-> 😳 **Running into problems?** Try searching for your error message on the [forums](https://forums.expo.io) &mdash; you're probably not the first person to encounter your issue, and the forums are a great resource for these types of problems.
```

Comment:
Why would you have problems installing Expo, `npm i -g expo-cli` isn't broken. This phrasings seemingly alludes to the opposite.

Reply:
I think it's not uncommon for windows users to run into permissions issues here, and we probably added this so people know they can search around for solutions rather than instantly thinking there's a bug with `expo-cli`

Reply:
but this line is probably enough https://github.com/expo/expo/pull/9998/files#diff-fcf7ec29a70a4d795b2a087f22bc5ff5R25